app/views/layouts/search-application.html.erb

```diff
@@ -1,27 +1,21 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
```

Comment:
Should we be removing the lang? Is this added back in when it's injected into the static template?

Reply:
Rails does it by magic.
Note it's still there on: http://finder-frontend-pr-318.herokuapp.com/search?q=test